scripts/download.sh

```
@@ -1 +1 @@
-wget -i ../pdf-extraction/data/pdfs.csv --wait=1 --tries=1 -P ../output
+wget -i ../pdf-extraction/data/pdfs.csv --wait=1 --tries=1 -P output/
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)